app/_src/using-mesh/managing-ingress-traffic/builtin.md

```diff
@@ -47,6 +48,29 @@ spec:
   tags:
     kuma.io/service: edge-gateway
 ```
+{% endif_version %}
+{% if_version gte:2.7.x %}
```

Comment:
does this being 3 times on the page warrant using a snippet and an `{% include` directive?

Reply:
done